package.json

```diff
@@ -97,34 +98,34 @@
     "markdown-link-check": "3.11.2",
     "mocha": "10.2.0",
     "named-html-entities-json": "1.0.0",
-    "prettier": "3.0.3",
     "svg-path-bbox": "1.2.5",
     "svg-path-segments": "1.0.0",
     "svglint": "2.4.0",
     "svgo": "3.0.2",
     "svgpath": "2.6.0",
-    "typescript": "5.2.2"
+    "typescript": "5.2.2",
+    "xo": "0.58.0"
   },
   "scripts": {
-    "build": "node scripts/build/package.js",
-    "clean": "node scripts/build/clean.js",
-    "format": "prettier --cache --write .",
+    "build": "./scripts/build/package.js",
+    "clean": "./scripts/build/clean.js",
+    "format": "prettier --cache --write '**/*.{md,json,yml}' && xo --fix",
     "lint": "npm run ourlint && npm run jslint && npm run jsonlint && npm run svglint && npm run wslint",
-    "ourlint": "node scripts/lint/ourlint.js",
-    "jslint": "prettier --cache --check .",
-    "jsonlint": "node scripts/lint/jsonlint.js",
-    "svglint": "svglint --ci $npm_config_icons",
+    "ourlint": "./scripts/lint/ourlint.js",
+    "jslint": "xo",
+    "jsonlint": "./scripts/lint/jsonlint.js",
+    "svglint": "svglint --ci $npm_config_icons --config svglint.config.mjs",
     "wslint": "editorconfig-checker",
     "prepare": "husky",
     "prepublishOnly": "npm run build",
     "postpublish": "npm run clean",
     "test": "mocha tests --reporter tests/min-reporter.cjs --inline-diffs",
     "pretest": "npm run prepublishOnly",
     "posttest": "npm run postpublish",
-    "get-filename": "node scripts/get-filename.js",
-    "add-icon-data": "node scripts/add-icon-data.js"
+    "get-filename": "./scripts/get-filename.js",
+    "add-icon-data": "./scripts/add-icon-data.js"
   },
   "engines": {
-    "node": ">=0.12.18"
```

Comment:
Sorry I think thay we are not in the same boat. This would make simple-icons to be uninstallable on Node.js versions prior to 12. Could you revert this change?

My suggested solution is to either:

a) Add a check to *sdk.mjs* to verify that imports of this file are being doing on Node.js >= 12.16.
b) Separate SDK on a different library.

Any of these can be done in another PR.

Reply:
np.